From: Heinz Mauelshagen <heinzm@redhat.com>
Date: Fri, 31 Oct 2008 14:40:42 +0100
Subject: [md] dm-raid45.c: add target to makefile
Message-id: 1225460442.1196.40.camel@o
O-Subject: [RHEL5.3 PATCH] dm-raid45.c: add target to makefile
Bugzilla: 437180
RH-Acked-by: Jonathan Brassow <jbrassow@redhat.com>
RH-Acked-by: Alasdair G Kergon <agk@redhat.com>
RH-Acked-by: Mikulas Patocka <mpatocka@redhat.com>

This patch adds a missing target to the Makefile to allow for dm-raid45
builds.

It adresses RHBZ 437180.

Heinz

diff --git a/drivers/md/Makefile b/drivers/md/Makefile
index eb99c74..31954ee 100644
--- a/drivers/md/Makefile
+++ b/drivers/md/Makefile
@@ -39,6 +39,8 @@ obj-$(CONFIG_DM_MULTIPATH_HP)	+= dm-hp-sw.o
 obj-$(CONFIG_DM_MULTIPATH_RDAC)	+= dm-rdac.o
 obj-$(CONFIG_DM_SNAPSHOT)	+= dm-snapshot.o
 obj-$(CONFIG_DM_MIRROR)		+= dm-mirror.o dm-log.o
+obj-$(CONFIG_DM_RAID45)		+= dm-raid45.o dm-log.o dm-mem-cache.o \
+				   dm-region_hash.o dm-message.o
 obj-$(CONFIG_DM_ZERO)		+= dm-zero.o
 
 quiet_cmd_unroll = UNROLL  $@
